CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 * None.
 
 ### Enhancements
-* None.
+* Support for experimental K2-compilation with `kotlin.experimental.tryK2=true`. (Issue [#1483](https://github.com/realm/realm-kotlin/issues/1483))
```

Comment:
Maybe add the restriction about companion classes and Serialization or maybe reference some other place (either issue or documentation about it)

Reply:
I created https://github.com/realm/realm-kotlin/issues/1567 to track it. This should enable users to find and workaround the issue. Do we also need it as known issues?

Reply:
Probably not, my guess is that more people will find it if we just pin #1567.